app/js/lib/foxbox/foxbox.js

```diff
@@ -123,49 +127,66 @@ export default class Foxbox extends Service {
    * @returns {Promise}
    * @private
    */
-  _initDiscovery() {
+  _initDiscovery(timeout) {
```

Comment:
__nit:__ maybe ```timeout = 0``` since you don't pass it when you call this method for the first time